### ISSUE TITLE
[Cub] Add a tutorial

### ISSUE THREAD

**Issue by mmaismma** [Hermit-Tools/Thumbnail-Maker#43]
## Description
Cub's Contraption has a very simple and self-explanatory UI. But it would be fashionable to have a tutorial thing.

## Possible solution(s)
Imagine a tutorial button on the webpage. The user clicks the button. A box shows up in the center of the screen. It shows an animation of step 1 i.e. "Click on the *Choose Background* button". The *next step* button (❯) appears when the animation ends. The same occurs for the other steps.

## Additional context
Even better if all animations are done by CSS not uploaded videos.

**Comment by mmaismma** [comment]
Even better if showing tutorial on pressing the **`F1`** key.

**Comment by shubhammehra4** [comment]
Hey, can I take on this?

**Comment by mmaismma** [comment]
@shubhammehra4 Why not? This is a pretty big task, feel free to contact me in case. Send a confirmatory message if you wish to work on this issue.

**Comment by shubhammehra4** [comment]
I'll be using Into.js
Can you please give me the detail of all the steps that I need to include in the tutorial

**Comment by mmaismma** [comment]
Sorry to inform you, but we do not allow the usage of libraries yet. If you wish to work with plain JS, inform me.

**Comment by shubhammehra4** [comment]
Is it okay if I use Bootstrap CSS & JS?

**Comment by mmaismma** [comment]
No, as it is also written in our contributing guidelines, we only allow usage of plain languages (something you can write without the need of linking any external file).
